aggregator/auth.go

```diff
@@ -185,3 +190,34 @@ func (r *RpcServer) verifyOperator(ctx context.Context, operatorAddr string) (bo
 
 	return auth.VerifyOperator(authRawHeaders[0], operatorAddr)
 }
+
+func (r *RpcServer) GetSignatureFormat(ctx context.Context, req interface{}) (interface{}, error) {
+	walletAddress := ""
+	if reqMap, ok := req.(map[string]interface{}); ok && reqMap["wallet"] != nil {
+		walletAddress = reqMap["wallet"].(string)
+	}
```

Comment:
We can throw and fail this function call if the `optiosn.wallet` input parameter is empty. Even better, could you add a ethereum wallet address check in addition of emptiness check here?